frontend/lib/icon/generated.tsx

```diff
@@ -7,12 +7,8 @@ export const IconArrowBlockUp = (props: React.SVGAttributes<SVGElement>) => (
 );
 
 export const IconArrowLeft = (props: React.SVGAttributes<SVGElement>) => (
-  <svg {...props} x="0px" y="0px" viewBox="0 0 24 24" role="img">
-    <path
-      d="M12,20c-0.3,0-0.5-0.1-0.7-0.3c-0.4-0.4-0.4-1,0-1.4l5.3-5.3H5c-0.6,0-1-0.4-1-1s0.4-1,1-1h11.6l-5.3-5.3
-	c-0.4-0.4-0.4-1,0-1.4s1-0.4,1.4,0l7,7c0.1,0.1,0.2,0.2,0.2,0.3c0,0.1,0.1,0.2,0.1,0.4l0,0c0,0,0,0,0,0l0,0c0,0.1,0,0.3-0.1,0.4
-	c0,0.1-0.1,0.2-0.2,0.3l-7,7C12.5,19.9,12.3,20,12,20z"
-    />
+  <svg {...props} viewBox="0 0 16 16" xmlns="http://www.w3.org/2000/svg" role="img">
+    <path d="M15 8H1M1 8L8 15M1 8L8 1" stroke-width="2" />
   </svg>
```

Comment:
The icons are generated from the `lib/icon/svg` list, please update the icon there and then run `npm run gen:icons` to update this file.

Reply:
Thanks - good to know :+1: